CHANGELOG.md

```diff
@@ -106,6 +106,11 @@
 
 ##### Enhancements
 
+* Add check to ignore urls in `line_length` rule when `ignores_urls`
```

Comment:
this is now in the wrong section since 0.15.0 has been released. please move up.